lab4_part2.py

```diff
@@ -0,0 +1,24 @@
+class SocialNetwork:
+    def __init__(self, name: str, user_count: int):
+        self.name = name
+        self.user_count = user_count
```

Comment:
Здесь и далее во всех функциях.
Добавить описание функций.
К примеру:
```suggestion
        """
        Инициализировать объект "Автомобиль"
        :param brand: марка автомобиля
        :param year_of_manufacture: год выпуска автомобиля
        :param color: цвет автомобиля
        :raise TypeError: если один из (brand, color) не является строкой или year_of_manufacture не является целым числом
        :raise ValueError: если year_of_manufacture меньше или равен 0
        """
```